### DISCUSSION TITLE
What can tgc detect from dukun langit and other scripts?

### DISCUSSION THREAD

**Discussion by TheLibur** [thatskymod/Sky-CotL-Scripts#99] [Q&A] [answered]
Hello, so after almost a year of being a legit sky player the daily cr grind has become super duper boring for me and went from daily fun with friends to an annoying chore for a cosmetic ill probably never even wear after a day, when my friend told me theres an application you can install that can insta cr you and generally help you with your daily sky routine i immediately downloaded but i didnt find any posts (sorry if i missed one) that says what tgc can and cant detect with the scripts, i value my account alot since i have put alot of effort into it and ik the risks involved when cheating so thats why i wanted to be safe and know which ones are the safest to use and to help any future readers with the same concern as me

**Comment by TrashPanda868** [comment, marked as answer]
> Hello, so after almost a year of being a legit sky player the daily cr grind has become super duper boring for me and went from daily fun with friends to an annoying chore for a cosmetic ill probably never even wear after a day, when my friend told me theres an application you can install that can insta cr you and generally help you with your daily sky routine i immediately downloaded but i didnt find any posts (sorry if i missed one) that says what tgc can and cant detect with the scripts, i value my account alot since i have put alot of effort into it and ik the risks involved when cheating so thats why i wanted to be safe and know which ones are the safest to use and to help any future readers with the same concern as me

Personally the way I see it nothing is really "safe" if you are doing something against any rules and that's the risk you gotta take. 
As a player who has been playing since Season of Abyss, I understand your pain and frustration. 
But I've been using Canvas for roughly 2 weeks now and haven't ran anything any problems or concerns for my account. 

But if you want some peace of mind, I have heard that TGC won't out right ban you, they will give you a warning and if you continue to ignore their warning and continue to cheat, then they will ban you. (I think it's a 14 day ban and then a permanent ban after that)

So I'm personally not the right person to get a good answer out of but I'm using it and still continue to use it without any problems. 

Just as long as you are being an idiot with it and showing it off to all your friends or telling your friends about it, I think you should be alright. 

I personally when I use it, find somewhere to hide and then collect my candles and stuff without prying eyes. It's just me being paranoid and it seems to be paying off so far. 

But all in all to answer your question, I use Roman and only Roman. I haven't used the other one but Roman seems the simplest. 

Just one click of a button and your done with each category. 

But all in all best of luck and I hope you come to a decision soon. 
TGC seems to like paying players and if you are a big spender (like me) they seem to ignore you, but that just how I see it, but no one is invulnerable really. 
Just don't get caught and play smart :)

**Reply by TheLibur** [reply]
does roman work on the live version? I read the readme file and it says its for sky beta i havent touched sky beta at all tbh so idk if you can cr there and itll count towards the public version, i went ahead and used dukun langit since it looked like the goto script for all your needs but it seems i downloaded the script at a bad time since im unable to teleport to wax due to the newest patch sky released.. 

i have read a another post here that if u use the insta cr thingy they can detect you so im making sure not to touch that part, the rest of the script just seems to be for funsies and lols that would definitely get you reported by strangers, but so far im really enjoying the one that instantly lights all the wax and the one that makes me immune to crabs and krills.

**Reply by TrashPanda868** [reply]
> > does roman work on the live version? I read the readme file and it says its for sky beta i havent touched sky beta at all tbh so idk if you can cr there and itll count towards the public version, i went ahead and used dukun langit since it looked like the goto script for all your needs but it seems i downloaded the script at a bad time since im unable to teleport to wax due to the newest patch sky released..
> 
> Yup Roman also works just fine for the live version. And I think/heard somewhere that the Sky beta and live version are two separate things, so candles don't stay the same when switching between the two

Correct. Live Sky and Beta Sky are two separate entities. You don't wanna be playing on beta if you wasn't invited by the devs. 
That would be a major red flag (I think) 

But personally I don't know anything about Dunkun Libar. I have been using Roman since the first time I used it and haven't had any issues or ran into any problems. 
Personally I think Roman is just a simplified version of Dukun. 

But like I said, I use Roman and use everything expect for the Insta WL and Insta Eden. 
I save that for reset.
